README.md

```diff
@@ -35,3 +35,19 @@ contributions"
 
 Ultimately, contributing guides should keep aligned with those suggested by
 FIWARE (see [here](https://github.com/Fiware/developmentGuidelines/blob/master/external_contributions.mediawiki)).
+
+### How to publish a new chart in the catalogue 
+Ultimately we will automate the process, but for the time being only owners of the repo will update the catalogue.
+
+The procedure is as follows, once the PR for an updated/new chart is approved:
+1. The owners will package the chart from the master repo:
+  ```
```

Comment:
the code block seems to be breaking the number sequence. try yet another nesting level